sdk/storage/storage-blob/test/aborter.spec.ts

```diff
@@ -8,6 +8,7 @@ import { ContainerClient } from "../src";
 import { getBSU, recorderEnvSetup } from "./utils";
 import { record, Recorder } from "@azure/test-utils-recorder";
 import * as dotenv from "dotenv";
+import { Context } from "mocha";
 dotenv.config();
 
 // tslint:disable:no-empty
```

Comment:
We don't use tslint anymore. Do you need this comment?

Reply:
Thanks for the reminding.. I've removed all tslint comments.